oreilly-scenarios/cockroach-with-friendpay/exploring-friendpay.md

```diff
@@ -0,0 +1,73 @@
+# Exploring FriendPay
+
+When we launched our instance of FriendPay, several tables were created to support the application. Let's explore some of these tables and their schemas.
+
+First, let's switch our database to the `friendpay` database we created earlier (when you click to execute the command, it will automatically open in the correct terminal):
+
+`USE friendpay;`{{execute T1}}
+
+List all the tables in the database:
+
+`SHOW TABLES;`{{execute T1}}
+
+We should see the following tables:
+
+* `Balances`: This table keeps the balances for users.
+* `SequelizeMeta`: This table stores metadata for the ORM we use in FriendPay and can be ignored.
+* `Transactions`: This table stores all transactions in FriendPay.
+* `Users`: This table stores all user information.
+
+## Create Users
+
+To get started, we need to create some users in FriendPay. Similar to before, commands will execute in the correct tab when you click on them. 
+
+This first request will create a user with the handle `kjackson`.
+
+`curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"firstName": "kevin", "lastName": "jackson", "email": "k.jackson@gmail.com", "handle":"kjackson"}'\
+  http://localhost:3000/createUser | jq`{{execute T3}}
+
+This will create another user with the handle `rwonder`:
+
+`curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"firstName": "robert", "lastName": "wonder", "email": "r.wonder@gmail.com", "handle":"rwonder"}'\
+  http://localhost:3000/createUser | jq`{{execute T3}}
```

Comment:
Have you explained what `jq` does? I know it's for manipulating JSON on the command line, just checking that the students have been told.

Reply:
Looks like it hasn't been explained yet. I'd recommend telling them something like:

```
To simulate the front-end, we're going to use `jq`, [a command-line tool for working with JSON](https://stedolan.github.io/jq/manual/).
```